xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp

```diff
@@ -44,15 +45,23 @@ using namespace PVR;
 #define SETTING_TMR_FIRST_DAY           "timer.firstday"
 #define SETTING_TMR_NAME                "timer.name"
 #define SETTING_TMR_DIR                 "timer.directory"
-#define SETTING_TMR_RADIO               "timer.radio"
 #define SETTING_TMR_CHNAME_RADIO        "timer.radiochannelname"
+#define SETTING_TMR_TYPE                "timer.type"
+#define SETTING_TMR_NEW_EPISODES        "timer.newepisodes"
+#define SETTING_TMR_BEGIN_PRE           "timer.startmargin"
+#define SETTING_TMR_END_POST            "timer.endmargin"
+
+#define OPENREADONLY 20
```

Comment:
Something a bit more descriptive might be nice, reading this I had no idea what this was for until I looked at its usage.